src/components/messenger/list/group-details-panel.tsx

```diff
@@ -56,7 +56,7 @@ export class GroupDetailsPanel extends React.Component<Properties, State> {
             <Input value={this.state.name} onChange={this.nameChanged} />
           </div>
 
-          <div>
+          <div className={c('field-info-container')}>
```

Comment:
Changes required here as the `ImageUpload` component design with error message resulted in changes to the positioning of the upload component in this group details panel.